code/src/main/webapp/JavaScript/script.js

```diff
@@ -119,25 +139,32 @@ var startApp = function() {
   gapi.load('auth2', function() {
     // Retrieve the singleton for the GoogleAuth library and set up the client.
     auth2 = gapi.auth2.init({
+      apiKey: 'AIzaSyBMrfBNGcVEtoRsoduXvYSjd9piD36W7Qg',
       client_id: '1080865471187-u1vse3ccv9te949244t9rngma01r226m.apps.googleusercontent.com',
-      cookiepolicy: 'single_host_origin',
-      // Request scopes in addition to 'profile' and 'email'
-      //scope: 'additional_scope'
+      scope: ''
+      //'https://www.googleapis.com/auth/calendar.events.readonly'
```

Comment:
Please remove this comment if it's not used.

Reply:
this code is from a different branch